tango/integrations/torch/train_config.py

```diff
@@ -74,17 +76,17 @@ class TrainConfig:
 
     log_every: int = 10
     """
-    Controls the frequency of log updates.
+    Controls the frequency of log updates, in number of optimizer steps
```

Comment:
😉
```suggestion
    Controls the frequency of log updates, in number of optimizer steps.
```

Reply:
It's not a complete sentence. No object, no period :-)

Reply:
As an American, I know nothing about grammar.